crates/loam-sdk/README.md

```diff
@@ -0,0 +1,121 @@
+# loam-sdk
+
+- [Getting Started](#getting-started)
+    - [Installation](#installation)
+    - [Setup](#setup)
+    - [Redeploy](#redeploy)
+- [Subcontracts](#subcontracts)
+    - [Creating Contract Subcontracts](#creating-contract-subcontracts)
+    - [External API](#external-api)
+- [CoreSubcontract](#coreriff)
+-   [Using the Core Subcontract](#using-the-coreriff)
+
+### Installation
+
+To install `just`, run the following command:
+
+```bash
+cargo install just
+```
+
+### Setup
+
+To set up the environment, run:
+
+```bash
+just setup
+```
+
+### Redeploy
+
+To see redeployment in action, use:
+
+```bash
+just redeploy
+```
+
+## Subcontracts
+
+A subcontract is a type that implements the `IntoKey` trait, which is used for lazily loading and storing the type.
+
+### Creating  Subcontracts
+
+Here's an example of how to create a contract riff:
```

Comment:
The word "riff" is used throughout this file. Let's update it all to "subcontract".